.github/workflows/pr-checks-for-qa-approvals.yml

```diff
@@ -0,0 +1,75 @@
+
+#
+# Checks if QA approvals are necessary, and if so runs the associated action when a new review is added
+#
+
+name: '[Utils] QA required approval'
+
+on:
+  pull_request:
+    types: [opened, edited, converted_to_draft, ready_for_review, reopened]
+    paths:
+      - 'VAMobile/src/**'
+      - 'VAMobile/package.json'
+      - '!VAMobile/src/**.test.*'
+  pull_request_review:
+    types: [submitted, dismissed]
+  pull_request_target:
+    branches:
+      - 'develop'
+  push: 
+    branches: 
+      - 'chanel-6261-qa-approval-workflow'     
+jobs:
+  check_for_qa_approval:
+    name: Requires QA approval
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check QA approval
+        
+        shell: bash
+        run: |
+          approvals=$(curl --request GET \
+          --url https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/reviews?per_page=100 \
+          --header 'Authorization: ${{ secrets.GITHUB_TOKEN }}' \
+          --header 'Content-Type: application/json' |
+          jq -c '[map(select(.state == "APPROVED")) | .[] .user.login]')
+
+          if [[ $(jq 'length' <<< "$approvals") -eq 0 ]]; then
+              echo 'No reviews found for this pull request.'
+              exit 1
+          fi
+
+          echo "${{secrets.GH_ACTIONS_PAT}}" >> token.txt
+          gh auth login --with-token < token.txt
+          required_approval_count=$(gh api \
+          -H "Accept: application/vnd.github+json" \
+          -H "X-GitHub-Api-Version: 2022-11-28" \
+          /repos/department-of-veterans-affairs/va-mobile-app/branches/develop/protection |
+          jq -c '.required_pull_request_reviews.required_approving_review_count')
+          echo "Approvers: $approvals"
+          
+          qa_team_members=$(
+            gh api \
+            -H "Accept: application/vnd.github+json" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            /orgs/department-of-veterans-affairs/teams/flagship-mobile-qa/members |
+            jq -c '[.[] | .login]'
+          )
+          echo "required approval count: $required_approval_count"
+          echo "QA team: $qa_team_members"
+          if [[ $(jq '. | length' <<< "$approvals") -gt $required_approval_count ]]
+            then
+              if [[ $(jq '[.[] | select(. | IN("timwright12", "rbontrager", "DJUltraTom", "TKDickson"))] | length' <<< "$approvals") -gt 0 ]]
```

Comment:
Should we be checking `$qa_team_members` instead of these hardcoded usernames? It doesn't seem like `$qa_team_members` is being used aside from the echo.

Reply:
we will be using hardcoded usernames. I removed $qateam_members

Reply:
Make sure this is noted somewhere in the QART documentation for onboarding/offboarding if we need to maintain this list. Can you spin up a ticket for post-MVP to use the team members in github (unless that's not possible)

Reply:
I tried to use qa team members, but it was just giving me hard time. I can look into it deeper on another ticket as well.

Reply:
Ticket has been created for post mvp